lms/djangoapps/teams/tests/test_models.py

```diff
@@ -335,5 +337,7 @@ def test_signals_course_context(self, signal_name):
         place in discussions outside of a team.
         """
         with self.assert_last_activity_updated(False):
-            signal = self.SIGNALS[signal_name]
-            signal.send(sender=None, user=self.user, post=self.mock_comment(context='course'))
+            with patch('lms.djangoapps.discussion.rest_api.tasks.send_response_notifications.apply_async'):
+                with patch('lms.djangoapps.discussion.rest_api.tasks.send_thread_created_notification.apply_async'):
+                    signal = self.SIGNALS[signal_name]
+                    signal.send(sender=None, user=self.user, post=self.mock_comment(context='course'))
```

Comment:
here also, can we check what `signal` hold, and what we expect?

Reply:
`SIGNALS` are all the signals that have been created in `django_comment_common`. These are using mocks, hence I don't think we need to test expected. Tasks output has their separate test cases.

FYI: These are existing test cases that need to be updated because of task.